app/controllers/application_controller.rb

```diff
@@ -1,3 +1,20 @@
 class ApplicationController < ActionController::Base
   allow_browser versions: :modern
+
+  def find_article_by_article_id
+    @article = Article.find(params[:article_id])
+  end
+
+  def find_article_by_id
+    @article = Article.find(params[:id])
+  end
+
+  def article_comments
+    return @article.comments
+  end
+
+  def org_user_article
+    return @organization.users.find_by(id: current_user.id).articles
+  end
```

Comment:
These are related to articles itself and not useful to others controller so Article Controller seems to be the best place for these 

In application controller we can have a code to find only the organization as organization needs to be used on every other controllers